app/tests/test_anti_virus_check.py

```diff
@@ -97,7 +103,10 @@ def test_send_for_av_scan_failure(self):
         responses.add(responses.POST, settings.ANTI_VIRUS_BASE_URL, json={'data_id': data_id}, status=200)
 
         responses.add(responses.GET, settings.ANTI_VIRUS_BASE_URL + "/" + data_id,
-                      json={'scan_results': {'progress_percentage': 100, 'scan_all_result_i': 1}}, status=200)
+                      json={
+                          'scan_results': {'scan_all_result_i': 0},
```

Comment:
Is that scan_all_result_i still going to be there?

Reply:
Yes the scan_results section is still available. The result will look like this: https://onlinehelp.opswat.com/corev4/Fetch_Scan_Result.html